### ISSUE TITLE
terraform init doesn't work with 1.0.3

### ISSUE THREAD

**Issue by mimozell** [CircleCI-Public/terraform-orb#47]
Error message: `Too many command line arguments. Did you mean to use -chdir?`
Usage:
```
- terraform/install:
    terraform_version: '1.0.3'
- terraform/init:
    path: environment/<<parameters.environment>>/stateful
```
![Screenshot 2021-07-29 at 13 15 13](https://user-images.githubusercontent.com/40452559/127482775-c82c1ba4-0777-4172-8fc6-880b310b7c60.png)

**Comment by neerajmandal** [comment]
I am experiencing the same issue with version 1.0.3.

**Comment by kenanbiel** [comment]
For some reason, this happens when you specify the version number during `terraform/init` step.

**Comment by kenanbiel** [comment]
Devs are now working on fix. I think it's not working for version 0.15.x and above https://github.com/CircleCI-Public/terraform-orb/issues/48

**Comment by punkdata** [comment]
@mimozell  @njm3754 @kenanbiel PR #46  should solve these compatibility issues for Terraform version 0.15.0 and above. I expect the merge to happen this week

**Comment by punkdata** [comment]
This should be fixed with the PR #46 merge. please use the new [TF Orb version 3.0.0](https://circleci.com/developer/orbs/orb/circleci/terraform?version=3.0.0)

Closing this PR
